tools/cli/commands/create.py

```diff
@@ -447,9 +447,56 @@ def create_repo(args, gcloud_repos, repo_name):
     Raises:
       subprocess.CalledProcessError: If the `gcloud` command fails
     """
+    if utils.print_info_messages(args):
+        print('Creating repo {0}'.format(repo_name))
+
     create_cmd = ['create', repo_name]
     utils.call_gcloud_quietly(args, gcloud_repos, create_cmd)
 
+    # Clone the repo into a local scratch directory
+    datalab_dir = tempfile.mkdtemp(prefix='datalab-')
+    clone_cmd = ['clone', repo_name, datalab_dir]
+    utils.call_gcloud_quietly(args, gcloud_repos, clone_cmd)
+
+    # Add a couple of files
+    if utils.print_info_messages(args):
+      print('Adding files to repo using local dir {0}'.format(datalab_dir))
+    with open(os.path.join(datalab_dir, '.gitignore'), 'w') as f:
+      f.write('.ipynb_checkpoints\n')
```

Comment:
.ipynb_checkpoints is a directory. This line matches the directory name, so everything under that directory gets ignored.